app/components/article-content.js

```diff
@@ -534,6 +538,10 @@ export default Component.extend(
 		handleCollapsibleSections() {
 			toArray(this.element.querySelectorAll('h2[section]'))
 				.forEach((header) => header.addEventListener('click', this.handleCollapsibleSectionHeaderClick.bind(this)));
+		},
+
+		uncollapseSections() {
+			toArray(this.element.querySelectorAll('h2[section]')).forEach((header) => header.classList.add('open-section'));
```

Comment:
you'll be missing calling this function
https://github.com/Wikia/mobile-wiki/pull/654/files#diff-93fe8a4709c07d310d26b67237a9cb5bR529
some things might not work as expected.

also I assume you're fine with jumping content, yes? I mean user will load an article and then after some time it'll expand.

Reply:
Thanks for the advice, I made some changes.
By default, wgVariable will be set to XX and for testing and analysing purposes on small percentage of users we're fine with that.